cron/main.go

```diff
@@ -84,6 +87,18 @@ func main() {
 	githubClient := github.NewClient(httpClient)
 	graphClient := githubv4.NewClient(httpClient)
 
+	exporter, err := monitoring.NewStackDriverExporter()
```

Comment:
rather than inlining this, would be good to have a separate function for this. Something like `startMetricsExporter()` which creates, starts, registers all views, and returns the exporter. Since we plan to add more views it'll get a bit messy here.

Reply:
Done.